### PR TITLE
made unused usings code fixer to consider all contexts

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

**Comment by heejaechang** [comment]
@mavasani @dpoeschl @srivatsn can you take a look? thank you

File: src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs [hunk 1]
```diff
@@ -163,7 +164,7 @@ private void RemoveUnusedImports(ITextBuffer subjectBuffer, CancellationToken ca
             var document = subjectBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
             if (document != null)
             {
-                var newDocument = RemoveUnnecessaryImportsService.RemoveUnnecessaryImportsAsync(document, cancellationToken).WaitAndGetResult(cancellationToken);
+                var newDocument = document.GetLanguageService<IRemoveUnnecessaryImportsService>().RemoveUnnecessaryImportsAsync(document, cancellationToken).WaitAndGetResult(cancellationToken);
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs]: Could this service be null?  Perhaps for a non-VB/C# document?

File: src/Features/VisualBasic/Portable/RemoveUnnecessaryImports/VisualBasicRemoveUnnecessaryImportsService.vb [hunk 2]
```diff
@@ -33,9 +32,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnnecessaryImports
...
         End Function
 
-        Public Function RemoveUnnecessaryImports(document As Document, model As SemanticModel, root As SyntaxNode, cancellationToken As CancellationToken) As Document Implements IRemoveUnnecessaryImportsService.RemoveUnnecessaryImports
+        Public Async Function RemoveUnnecessaryImportsAsync(document As Document, cancellationToken As CancellationToken) As Task(Of Document) Implements IRemoveUnnecessaryImportsService.RemoveUnnecessaryImportsAsync
             Using Logger.LogBlock(FunctionId.Refactoring_RemoveUnnecessaryImports_VisualBasic, cancellationToken)
-                Dim unnecessaryImports = DirectCast(GetIndividualUnnecessaryImports(model, root, cancellationToken), ISet(Of ImportsClauseSyntax))
+                Dim model = Await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(False)
+                Dim root = Await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(False)
+
+                Dim unnecessaryImports = New HashSet(Of ImportsClauseSyntax)(GetIndividualUnnecessaryImportsOrEmpty(model, root, cancellationToken), Me)
```
Comment (CyrusNajmabadi) [inline src/Features/VisualBasic/Portable/RemoveUnnecessaryImports/VisualBasicRemoveUnnecessaryImportsService.vb]: This code looks identical.  It feels like the VBRemoveUnnecessaryService and the CSharpRemoveUnnecessaryService should share an abstract base class to remove duplicated logic that we have to keep in sync.

File: src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs (hunk 1 above)
Comment (heejaechang) [inline src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs]: no, the handler is set to CSharp and VB content type. so if this service doesnt exist, VS won't be in a working state anyway.

**Comment by heejaechang** [comment]
@dotnet-bot retest win eta please

**Comment by heejaechang** [comment]
test win eta please

**Comment by heejaechang** [comment]
@dotnet-bot test win eta please

**Comment by heejaechang** [comment]
@dotnet-bot retest win please

**Comment by heejaechang** [comment]
@dotnet-bot retest this please

**Pull request merged**